### PR TITLE
Update after pip internals changes

### PR DESCRIPTION
**Pull request by fbochu**

### CODE REVIEW

File: src/grocker/six.py [hunk 1]
```diff
@@ -40,3 +41,15 @@ def __enter__(self):
...
+
+
+def getstatusoutput(cmd):
+    try:
+        data = subprocess.check_output(cmd, shell=True, universal_newlines=True, stderr=subprocess.STDOUT)
+        exitcode = 0
+    except subprocess.CalledProcessError as ex:
+        data = ex.output
+        exitcode = ex.returncode
+    if data[-1:] == '\n':
```
Comment (xavfernandez) [inline src/grocker/six.py]: `data = subprocess.check_output(cmd, universal_newlines=True, stderr=subprocess.STDOUT).strip()` should do the trick

File: src/grocker/helpers.py [hunk 2]
```diff
@@ -72,7 +74,12 @@ def pip_conf(pip_conf_path=None):
     """
     if pip_conf_path is None or not os.path.exists(pip_conf_path):
         with tempfile.NamedTemporaryFile('w', dir=os.path.expanduser('~/.cache')) as f:
-            config = pip.baseparser.ConfigOptionParser(name='global').config
+            config = configparser.RawConfigParser()
+            config.add_section('global')
+            for key in ['index-url', 'timeout', 'extra-index']:
+                exit_code, value = six.getstatusoutput('pip config get global.{}'.format(key))
+                if exit_code:
```
Comment (xavfernandez) [inline src/grocker/helpers.py]: `if not exit_code:` ?

File: src/grocker/six.py [hunk 3]
```diff
@@ -40,3 +41,15 @@ def __enter__(self):
 
     def __exit__(self, exc, _value, _tb):
         shutil.rmtree(self.name)
+
+
+def getstatusoutput(cmd):
+    try:
+        data = subprocess.check_output(cmd, shell=True, universal_newlines=True, stderr=subprocess.STDOUT)
```
Comment (xavfernandez) [inline src/grocker/six.py]: No need for `shell=True` ?

File: src/grocker/six.py (hunk 1 above)
Comment (fbochu) [reply src/grocker/six.py]: It's a duplication of https://github.com/python/cpython/blob/3.6/Lib/subprocess.py#L495 for compatibility purpose (Python 2.7). But according to the documentation it's a legacy function, maybe I should replace it by an another subprocess function.

**Pull request merged**